.claude/commands/red.md

```
@@ -0,0 +1,111 @@
+---
+description: Execute TDD Red Phase - write ONE failing test
+argument-hint: [optional additional info]
+---
+
+**User arguments:**
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)